specification/cognitiveservices/data-plane/InkRecognizer/preview/v1.0/InkRecognizer.json

```diff
@@ -292,6 +292,94 @@
+          },
+          "barrelPressure" : {
+            "type" : "number",
+            "description" : "The force exerted directly by the user on a transducer sensor, such as a pressure-sensitive button on the barrel of a stylus. This may not be used for recognition."
+          },
+          "timestamp" : {
+            "type" : "number",
+            "description" : "The time relative to the absolute time the transducer last became active. This may not be used for recognition."
+          },
+          "xTilt" : {
+            "type" : "number",
+            "description" : "The plane angle between the Y-Z plane and the plane containing the transducer axis and the Y axis. This may not be used for recognition."
+          },
+          "yTilt" : {
+            "type" : "number",
+            "description" : "The angle between the X-Z and transducer-X planes. A positive Y Tilt is toward the user. This may not be used for recognition."
+          },
+          "width" : {
+            "type" : "number",
+            "description" : "The width of the tip of the writing instrument. This is used by touch screen devices to report the width of the finger contact on the writing surface. This may not be used for recognition."
+          },
+          "height" : {
+            "type" : "number",
+            "description" : "The height of the tip of the writing instrument. This is used by touch screen devices to report the height of the finger contact on the writing surface. This may not be used for recognition."
+          },
+          "tipSwitch" : {
+            "type" : "boolean",
+            "description" : "A switch located on the tip of a stylus indicating contact of the stylus with a surface. This may not be used for recognition."
+          },
+          "inverted" : {
+            "type" : "boolean",
+            "description" : "A value that indicates that the currently sensed position originates from the end of a stylus opposite the tip switch. This may not be used for recognition."
+          },
+          "barrelSwitch" : {
+            "type" : "boolean",
+            "description" : "A non-tip button located on the barrel of a stylus. Its function is typically mapped to a system secondary button. This may not be used for recognition."
+          },
+          "eraser" : {
+            "type" : "boolean",
+            "description" : "The control is used for erasing objects. It is typically located opposite the writing end of a stylus. This may not be used for recognition."
+          },
+          "secondaryTip" : {
+            "type" : "boolean",
+            "description" : "A secondary switch used in conjunction with the tip switch to indicate pressure above a certain threshold applied with the stylus. This may not be used for recognition."
+          }
+        },
+        "required" : [
```

Comment:
Are there `default` values for any of the optional attributes? If so, it would be great to indicate what they are.

Reply:
There are no defaults for any of the attributes. They're supposed to be values reported by pen/touch hardware based on the current user activity.